### DISCUSSION TITLE
Can one use `mockserver` to mock stripe too

### DISCUSSION THREAD

**Discussion by kasir-barati** [airbytehq/airbyte#45070] [General]
I know that this might be out of place but I wanted to know if it is possible for me to mock stripe with `mockserver`. I read this and I know that it is possible for the server but I am not sure about stripe.

https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/README.md#when-you-dont-have-access-to-the-api

**Comment by natikgadzhi** [comment]
I don't remember from the top of my head if it's the absolute same or not, BUT, look at `source-stripe` unit tests, there's a directory with tests that mock Stripe responses. Sounds like something very close to what you want.

@maxi297 made that framework! 👏

**Reply by kasir-barati** [reply]
Thanks for the reply. I did manage to do it. Here is a link to it: https://dev.to/kasir-barati/test-stripe-with-mockserver-57mo

In that post you can find the necessary information about how I did it.

I sincerely thank you @natikgadzhi :heart:.

[a comment by an automated account is not shown]
